parsl/providers/pbspro/pbspro.py

```diff
@@ -119,6 +124,12 @@ def submit(self, command, tasks_per_node, job_name="parsl"):
         job_config["worker_init"] = self.worker_init
         job_config["user_script"] = command
 
+        # Add a colon to select_options if one isn't included
+        if self.select_options and not self.select_options.startswith(":"):
```

Comment:
if I put a `:` on the front, this silently ignores the options rather than erroring?

Reply:
oh ignore this i misunderstood